lib/trino-plugin-toolkit/src/main/java/io/trino/plugin/base/session/PropertyMetadataUtil.java

```diff
@@ -25,20 +26,25 @@
 {
     private PropertyMetadataUtil() {}
 
-    public static PropertyMetadata<DataSize> dataSizeProperty(String name, String description, DataSize defaultValue, boolean hidden)
+    public static PropertyMetadata<DataSize> dataSizeProperty(String name, String description, DataSize defaultValue)
     {
-        return dataSizeProperty(name, description, defaultValue, value -> {}, hidden);
+        return dataSizeProperty(name, description, defaultValue, value -> {}, EnumSet.noneOf(PropertyMetadata.Flag.class));
```

Comment:
static import `EnumSet.noneOf`

Reply:
acutally do not. Just `noneOf` is not explanatory enough. If sth define `NO_FLAGS` constant in PropertyMetadata and use here.